queries/nim/highlights.scm

```diff
@@ -0,0 +1,35 @@
+(keyw) @keyword
+
+[
+ "["
+ "]"
+ "("
+ ")"
+ ] @punctuation.bracket
+
+[
+ "."
+ ":"
+ ] @punctuation.delimiter
+
+[
+  (operator)
+  (opr)
+  "="
+] @operator
+
+(importStmt (keyw) @include)
+(fromStmt (keyw) @include)
+(str_lit) @string
+(int_lit) @number
+[(float_lit) (float_suffix)] @float
+
+(comment) @comment
```

Comment:
I think you should also add
`(docComment) @comment`

Reply:
Actually maybe all of these
![image](https://user-images.githubusercontent.com/22427111/192083684-2f96c75b-f026-474b-98f5-f8e797a16389.png)

Reply:
Thank you! I integrated your suggestion!